test/services/HolderOfAuthorisationCodeSpec.scala

```diff
@@ -41,5 +41,11 @@ class HolderOfAuthorisationCodeSpec extends UnitSpec {
       val codes = HolderOfAuthorisationCode.all.filter(code => Set("ACE", "UKCS").contains(code.value))
       codes mustBe List(HolderOfAuthorisationCode("ACE"), HolderOfAuthorisationCode("UKCS"))
     }
+
+    "contain values for CEDS-1773" in {
```

Comment:
I think it should be differen name for the test

Reply:
Got any suggestions?